sasstests.py

```diff
@@ -710,6 +711,17 @@ def test_successful(self):
             self.assertEqual(contentsf1, 'a b {\n  width: 100%; }\n')
             self.assertEqual(contentsf2, 'foo {\n  width: 100%; }\n')
 
+    def test_compile_directories_unicode(self):
+        with tempdir() as tmpdir:
+            input_dir = os.path.join(tmpdir, 'input')
+            output_dir = os.path.join(tmpdir, 'output')
+            os.makedirs(input_dir)
+            with io.open(os.path.join(input_dir, 'test.scss'), 'w') as f:
+                f.write(u'a { content: "☃"; }')
+            # Raised a UnicodeEncodeError in py2 before #82 (issue #72)
```

Comment:
ah right, because in python3 `open` acts like `io.open(...)` without an encoding (choosing the os default).  So broken in py2 for non-ascii and in py3 when the default encoding is not able to encode things.